lib/npm.js

```diff
@@ -48,15 +48,14 @@ class Npm {
   #logChalk = null
   #noColorChalk = null
 
-  #outputBuffer = []
+  #display = null
   #logFile = new LogFile()
-  #display = new Display()
   #timers = new Timers({
     start: 'npm',
     listener: (name, ms) => {
-      const args = ['timing', name, `Completed in ${ms}ms`]
-      this.#logFile.log(...args)
-      this.#display.log(...args)
+      const args = [name, `Completed in ${ms}ms`]
+      this.#logFile.log('timing', ...args)
+      this.#display.logTiming(...args)
```

Comment:
This decision seems odd to me at first glance.

Reply:
I did this because I wanted to make it clear that the only public method here is to add a timing log. There's no other public method to log something. You have to emit with proc log. But proc-log doesn't do timing. Maybe we should add that instead?

Reply:
Yeah that makes sense.  npm can still be the source of truth on when that actually gets displayed/logged to the file like it does now.